src/normalizer/mod.rs

```diff
@@ -71,10 +64,8 @@ pub struct NormalizerOption {
     pub create_char_map: bool,
 }
 
-impl NormalizerOption {
-    
-    /// Creates a new [`NormalizerOption`] with default options set.
-    pub fn new() -> NormalizerOption {
+impl Default for NormalizerOption {
+    fn default() -> Self {
         NormalizerOption {
             create_char_map: true,
```

Comment:
Note that I changed the definition of the `test_normalizer!` macro by passing `NormalizerOption { create_char_map: true }` to `normalize()` because several of the tests expect a char map to be created.